tests/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -0,0 +1,68 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import AlternativeVerticalsComponent from '../../../../src/ui/components/results/alternativeverticalscomponent';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig, storage;
+
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  COMPONENT_MANAGER.getComponentNamesForComponentTypes = () => [];
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+  storage = COMPONENT_MANAGER.core.storage;
+  delete global.window.location;
+  global.window.location = {
+    search: '?query=office+space'
+  };
+
+  defaultConfig = {
+    container: '#test-component',
+    verticalsConfig: [
+      {
+        label: 'Home',
+        url: 'index.html'
+      },
+      {
+        verticalKey: 'KM',
+        label: 'Locations',
+        url: 'locations.html'
+      }
+    ],
+    baseUniversalUrl: './index.html?query=office+space',
```

Comment:
Should `baseUniversalUrl` have the `query` param?

Reply:
currently VerticalResults DOES pass query params down, but the AlternativeVerticals component doesn't require you do this so I removed the `query` param from the test